opus/application/static_media/css/opus.css

```diff
@@ -703,6 +703,15 @@ button.op-mini-thumbnail-zoom {
     right: 1.25em;
 }
 
+/* Make sure the a tags for categories take up full width in select metadata
+and search sidebar menu */
+a[href^="#selector-submenu-search"], a[href^="#selector-submenu-obs"],
```

Comment:
Would it be better to add a class to these instead of using the ID?

Reply:
Added a class "op-submenu-category" to these anchor tags.